solveengine/examples/usage.py

```diff
@@ -8,7 +8,7 @@
 #
 # the token
 token = "2prMEegqCAFB5eqmfuXmGufyJ+PcMzJbZaQcvqrhtx4="
-staticmethod
+
```

Comment:
that's weird, it should have been removed, I'll double check for the next commit